R/explanation.R

```diff
@@ -16,11 +16,11 @@
 #'
 #' @param ... Additional arguments passed to \code{\link{prepare_data}}
 #'
-#' @details The most important thing to notice is that \code{shapr} has implemented three different
+#' @details The most important thing to notice is that \code{shapr} has implemented four different
 #' approaches for estimating the conditional distributions of the data, namely \code{"empirical"},
-#' \code{"gaussian"} and \code{"copula"}.
+#' \code{"gaussian"}, \code{"copula"} and \code{"ctree"}.
 #'
-#' In addition to this the user will also have the option of combining the three approaches.
+#' In addition to this the user will also have the option of combining the four approaches.
```

Comment:
In addition, the user also has the option of combining the four approaches.
```suggestion
#' In addition to this the user will also have the option of combining the four approaches.
```